app/src/main/java/com/gruutnetworks/gruutsigner/util/AuthCertUtil.java

```diff
@@ -250,7 +348,7 @@ public String signData(byte[] data)
      * @param certification The certification stored in the Android Keystore
      * @return A boolean value telling you whether the signature is valid or not.
      */
-    public boolean verifyData(byte[] input, String signatureStr, String certification)
+    private boolean verifyData(byte[] input, String signatureStr, String certification)
```

Comment:
```suggestion
    private boolean verifySender(byte[] input, String signatureStr, String certification)
```
타인의 인증서로 확인하는 거죠? 머저가 내가 아는 머저가 보낸게 맞는지.
그럼 발신자를 검증한다는 의미로 이름 짓는게 어떨까요?